packages/amplify-graphql-schema-generator/src/__tests__/generate-ts-data-schema.test.ts

```diff
@@ -249,6 +249,108 @@ describe('Type name conversions', () => {
     expect(graphqlSchema).toMatchSnapshot();
   });
 
+  it.each([
```

Comment:
These cases are not exhaustive over every `Field` that could exist but cover the most unique cases.